src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -75,11 +74,12 @@ impl JoinRow {
         for v in &self.row.0 {
             vec.extend_from_slice(&serialize_cell(v)?)
         }
-        let mut serializer = memcomparable::Serializer::new(vec![]);
 
         // Serialize degree.
-        self.degree.serialize(&mut serializer)?;
-        vec.extend_from_slice(&serializer.into_inner());
+        let mut degree_buf: Vec<u8> = vec![];
+        degree_buf.put_u64_le(self.degree);
+        vec.extend_from_slice(&degree_buf);
```

Comment:
`vec.extend_from_slice(&self.degree.to_le_bytes())` should be enough?

Reply:
ditto

Reply:
Emmm didnt notice the auto merge

Reply:
Ah sorry, will fix those later.